styles.css

```diff
@@ -0,0 +1,31 @@
+.artboard {
+    display: grid;
+    grid-template-columns: auto auto;
+    justify-content: center;
+    column-gap: 24px;
+    row-gap: 24px;
+    margin: 36px;
+}
+
+.artboard a {
+    background-color: red; /* remove later and replace with background-image */
```

Comment:
I would make this change now or remove it before merging the code in.

Reply:
Temporary code has been removed from the pre-alpha code so that it will be apart of the production and not a part of the production